src/logger/logger.types.ts

```diff
@@ -0,0 +1,13 @@
+export type Loggable = string | Record<string, unknown>
+
+export interface LogMethod {
+  (message: Loggable): void
+}
+
+export type Formatter = (message: string) => string
+
+export interface Logger {
+  info: LogMethod
```

Comment:
can we add debug & fatal as well?

Reply:
sure, but i do have a few questions:

1. should any of those be optional? we might wish to make debug optional cos prod/staging loggers might not require those (but take note, also, that `winston-cloudwatch` has `debug` but not `fatal`)
2. what colour should we use? thinking blue for debug, bg red for fatal

Reply:
> should any of those be optional?
Debug & fatal can be both optional

> what colour should we use on console logger? thinking blue for debug, bg red for fatal
yea that's what we usually use, sounds good!